frontend/src/components/utils/Gas.tsx

```diff
@@ -0,0 +1,22 @@
+import BN from "bn.js";
+
+interface Gas {
+  gas: BN;
+}
+
+const MGAS = new BN(10 ** 6);
+const TGAS = new BN(10 ** 12);
+
+export default ({ gas }: Gas) => {
```

Comment:
Since it is a React component, we, conventionally, use `Props` as the type name.